src/services/fetchers/supervielle.ts

```diff
@@ -25,13 +25,15 @@ class Supervielle {
 
       return {
         name: 'Supervielle',
+        url: 'https://bit.ly/2TfvDaK',
```

Comment:
@SantiDepetris 
this link is pointing to `empresas homebanking` we should point out https://personas.supervielle.com.ar/

Reply:
Done